version.json

```diff
@@ -0,0 +1,3 @@
+{
+    "version": "1.4.0"
+}
```

Comment:
Do you know why my screen has circuit with "-" inside in red?

Reply:
Not sure. I dont see anything red on my screen